src/main/java/com/paskar/email/application/console/InformationFromConsole.java

```diff
@@ -0,0 +1,117 @@
+package com.paskar.email.application.console;
+
+
+import lombok.Getter;
+
+import java.io.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class InformationFromConsole implements Serializable {
+
+    @Getter
+    private List<Email> listWithAllEmails = new ArrayList<>();
+
+    private static final long serialVersionUID = 1L;
+
+    private transient static final DateTimeFormatter FORMATTER =
+            DateTimeFormatter.ofPattern("MM dd yyyy HH:mm");
+    private transient static final BufferedReader READER = new BufferedReader(new InputStreamReader(System.in));
+
+
+    public static void main(String[] args) throws IOException {
```

Comment:
We should have only one entry point inside application and it's should be the one with SpringApplication.run method. Consider making this class a spring bean and implement CommandLineRunner to be able to process input from the user